src/cascadia/TerminalApp/JsonUtilsNew.h

```diff
@@ -248,7 +277,7 @@ namespace TerminalApp::JsonUtils
             }
 
             const auto string{ Detail::GetStringView(json) };
-            return (string.length() == 7 || string.length() == 3) && string.front() == '#';
+            return (string.length() == 7 || string.length() == 4) && string.front() == '#';

```

Comment:
Curious: what happened here?

Reply:
For an `#rgb` string, `string.length() == 4`, not 3. This must've been an oversight